phylogenetic/README.md

```diff
@@ -9,6 +9,20 @@ Follow the [standard installation instructions](https://docs.nextstrain.org/en/l
 
 ## Usage
 
+### Example build
+
+You can run an example build using the example data provided in this repository via:
+
+```
+nextstrain build .  --configfile profiles/ci/builds.yaml
+```
+
+When the build has finished running, view the output Auspice trees via:
+
+```
+nextstrain view auspice/
```

Comment:
_minor_

I'd suggest this be `nextstrain view .` for consistency with `nextstrain build .` above.

Reply:
Ah, I keep forgetting that `nextstrain view .` works!